web-server/routes/prosumer.js

```diff
@@ -210,6 +210,19 @@ router.post('/market/price',
             marketController.getPrice);
 
 
+
+/**
+ * GET request /prosumer/climate/wind for getting wind speed
+ */
+router.get('/climate/wind', prosumerController.getWindSpeed);
```

Comment:
```suggestion
router.get('/climate/wind', auth.verify, prosumerController.getWindSpeed);
```

Reply:
Wind is public information there is no need to authenticate